README.md

```diff
@@ -31,7 +31,7 @@ database. These configurations are added to the `persistence.xml` file.
 2. Create a file named `persistence.xml` in the `META-INF` folder.
 3. Your project directory structure should look like the following:  
 
-![project structure persistence.xml](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-jpa/persistence_xml.png)  
+![persistence.xml](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-jpa/persistence_xml_project.png)  
```

Comment:
These two images look the same? Was that on purpose??

Reply:
intellij sometimes won't refresh the image in the preview, so renaming forces it.

I updated the images to crop out the project path.

Reply:
If you don't want to rename it, you can try this trick that Liz sent to me a while ago:

curl -X PURGE {url of cached image}

Source: [from Liz](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-anonymized-urls#removing-an-image-from-camos-cache). "That caching issue is a real pita. It took me a long time to figure out what was happening. It’s GitHub that does it, I believe, and it’s incredibly stubborn about it. The purge should do the trick for you. The cache will eventually get updated, but to see results right away, try the purge"

Reply:
cool, thanks!